gulp.d/tasks/build.js

```diff
@@ -129,24 +126,28 @@ module.exports = (src, dest, preview) => () => {
       .src(['css/site.css', 'css/vendor/*.css'], { ...opts, sourcemaps })
       .pipe(postcss((file) => ({ plugins: postcssPlugins, options: { file } }))),
     vfs.src('font/*.{ttf,woff*(2)}', opts),
-    vfs.src('img/**/*.{gif,ico,jpg,png,svg}', opts).pipe(
-      preview
-        ? through()
-        : imagemin(
-          [
-            imagemin.gifsicle(),
-            imagemin.jpegtran(),
-            imagemin.optipng(),
-            imagemin.svgo({
-              plugins: [
-                { cleanupIDs: { preservePrefixes: ['icon-', 'view-'] } },
-                { removeViewBox: false },
-                { removeDesc: false },
-              ],
-            }),
-          ].reduce((accum, it) => (it ? accum.concat(it) : accum), [])
-        )
-    ),
+    vfs.src('img/**/*.{gif,ico,jpg,png,svg}', opts)
+      .pipe(map((file, enc, next) => {
+        if (!preview) {
+          import('gulp-imagemin').then((imagemin) => {
```

Comment:
version 8+ of `gulp-imagemin` only works with ESM
```
❯ ./node_modules/gulp/bin/gulp.js bundle
Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/rachel/sandbox/antora-ui/node_modules/gulp-imagemin/index.js from /Users/rachel/sandbox/antora-ui/gulp.d/tasks/build.js not supported.
Instead change the require of index.js in /Users/rachel/sandbox/antora-ui/gulp.d/tasks/build.js to a dynamic import() which is available in all CommonJS modules.
    at Object.<anonymous> (/Users/rachel/sandbox/antora-ui/gulp.d/tasks/build.js:9:18)
    at /Users/rachel/sandbox/antora-ui/node_modules/require-directory/index.js:76:17
    at Array.forEach (<anonymous>)
    at requireDirectory (/Users/rachel/sandbox/antora-ui/node_modules/require-directory/index.js:59:24)
    at Object.<anonymous> (/Users/rachel/sandbox/antora-ui/gulp.d/tasks/index.js:5:46)
    at Object.<anonymous> (/Users/rachel/sandbox/antora-ui/gulpfile.js:17:14)
    at async Promise.all (index 0) {
  code: 'ERR_REQUIRE_ESM'
}
```
switch to dynamic import instead